client/lib/plans/index.js

```diff
@@ -28,3 +28,7 @@ export function getDaysUntilExpiry( plan ) {
 export function isInGracePeriod( plan ) {
 	return getDaysUntilUserFacingExpiry( plan ) <= 0;
 };
+
+export function isInGracePeriod( plan ) {
```

Comment:
Yeah, that function was added in two PRs at the same time and was duplicated during the rebase. Nice catch – I removed it.